src/main/java/com/management/servicebe/search/ui/SearchRestController.java

```diff
@@ -0,0 +1,27 @@
+package com.management.servicebe.search.ui;
+
+import com.management.servicebe.search.application.SearchFoodService;
+import com.management.servicebe.search.ui.response.SearchResponseDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class SearchRestController {
+
+	private final SearchFoodService searchFoodService;
+
+	@GetMapping("/food")
+	public SearchResponseDto searchFood(
+			@RequestParam String searchWord,
```

Comment:
`searchWord` 보다 `keyword`나 `searchedWord`가 어떨까요?
```suggestion
			@RequestParam String searchedWord,
```

Reply:
단순 "검색어" 라는걸 번역했을때 searchWord가 나오더군요 keyword보다는 searchedWord가 나은것같긴한데 저는  searchWord나 searchedWord 중에서는 크게 상관없습니다. searchedWord로 변경할까요?

Reply:
오 그렇네요! 말씀하신 것처럼 `searchWord`가 더 의미있군요! 좋습니다 👍